scripts/train_lm.py

```diff
@@ -296,6 +296,9 @@ def CheckFreshness(tgt_file, ref_files):
 
 # get ngram counts
 lm_name = vocab_name + '_' + str(args.order)
+if args.min_counts != '':
+    # replace '=' to '@', since '=' need to be escaped in shell
+    lm_name += '_' + '-'.join(args.min_counts.replace('=', '@').split())
```

Comment:
I think if you did
`   lm_name += '_' + '_'.join(args.min_counts.replace('=', '-').split()) `
the resulting names would be less scary looking... I don't want people to be put off by at-signs in filenames.